<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>669</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
      <item>
       <widget class="QWidget" name="widget_caculator" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="et_formula">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_enter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Enter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="0">
            <widget class="QPushButton" name="btn_pi">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>pi</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_log">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>log</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="btn_ce">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>CE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_atan">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>atan</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btn_tan">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>tan</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_e">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>e</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="btn_ln">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ln</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_sin">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>sin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_cos">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>cos</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_acos">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>acos</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_toRad">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>toRad</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_asin">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>asin</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tb_resultHistory"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_emptyList">
             <property name="text">
              <string>Empty List</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_resetEngine">
             <property name="text">
              <string>Reset JS Engine</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_functionEditor">
             <property name="text">
              <string>Function Editor</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_scriptEditor" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="layout_script">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Function Editor:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_scriptEnter">
           <property name="font">
            <font>
             <family>Adobe Arabic</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadJSFiles"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelpFunction"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadJSFiles">
   <property name="text">
    <string>Load .js File</string>
   </property>
  </action>
  <action name="actionHelpFunction">
   <property name="text">
    <string>About define function</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
